subgraphs/curve-finance/src/services/pools.ts

```diff
@@ -70,7 +70,8 @@ export function createNewPool(
   pool.gauge = gaugeAddress.toHexString();
   const inputTokens = getPoolCoins(pool);
   pool.coins = inputTokens;
-  pool.inputTokens = inputTokens.sort();
+  pool.inputTokens = inputTokens;
```

Comment:
Is this the only change? If so, can you revert all the other changes (including the commented line below)?